repo/test/test_repo.go

```diff
@@ -95,7 +95,7 @@ func newTestFS(ctx context.Context) *muxfs.Mux {
 
 // NewTestRepo generates a repository usable for testing purposes
 func NewTestRepo() (mr *repo.MemRepo, err error) {
-	datasets := []string{"movies", "cities", "counter", "craigslist", "sitemap"}
+	datasets := []string{"movies", "movies_directors", "cities", "counter", "craigslist", "sitemap"}
```

Comment:
This breaks many of the tests in this codebase. You can run `go test ./...` to run all tests; they should pass.

The reason this doesn't work is that many of our older tests use `NewTestRepo`, and changing the list of datasets here will modify IPFS hashes that those tests expect to see. That those tests are fragile isn't a good thing, but they still need to pass.

If what you're trying to do if have two datasets with joinable columns, you should just create datasets needed for your new test within that test itself. Our `TestRunner` pattern exists in a few different places, and handles this: it creates an empty repo in a temp directory and can create directories that are cleaned up at destruction time. See `MustSaveFromBody` in https://github.com/qri-io/qri/blob/master/lib/test_runner_test.go#L146. Unfortunately the `lib` version of `TestRunner` is different from `sql`'s, so you'll need to copy enough functionality from there to here.